src/lib/broadcast.jl

```diff
@@ -89,11 +88,6 @@ end
 
 @adjoint broadcasted(::typeof(identity), x::Numeric) = x, Δ -> (nothing, Δ)
 
-@adjoint function broadcasted(::typeof(σ), x::Numeric)
```

Comment:
Should rules like this move?

If I understand right, these break broadcast fusion, but are still worthwhile because Zygote's generic broadcast rule is pretty slow. But that's specific to Zygote.

Or perhaps, they should move to be ZygoteRules rules located within NNlib.

Reply:
Do you mean whether to keep them at all? I moved these to ChainRulesCore as well, I don't think we need ZygoteRules for these.

Reply:
What I mean is that they might not be desirable for other AD packages besides Zygote. With a better handling of broadcasting in general, using just the scalar rule for `σ` might be more efficient than using this one. Perhaps they should be thought of as performance patches for Zygote specifically, not mathematical properties for general use.

Reply:
Ah, that's a good point! Maybe we still want to use ZygoteRules for these then.

Reply:
Addressed in FluxML/NNlib.jl#242